docs/emails.rst

```diff
@@ -281,3 +293,14 @@ Store e-mail body in a HTML file is better from code readability. Therefore this
 ^^^^^^^^^^^^^^^
 
 E-mail body can be changed in the database therefore reverse operation to ``sync_emails`` can be done with this command. You must select directory where e-mails body in HTML format will be stored.
+
+
+``pull_emails_info``
+^^^^^^^^^^^^^^^^^^^^
+
+Synchronize e-mail message status from the provider.
+
+Webhooks
+--------
+
+Mandril provides notifications system which notify your URL endpoint that some message status was changed. For this purpose pymess provides view ``pymess.webhooks.mandrill.MandrillWebhookView`` which you only adds to your ``django urls``. Every notification will mark message to be updated with the ``pull_emails_info`` command.
```

Comment:
- Mandrill
- notification system
- notifies your URL endpoint
- which you simply add to your